### DISCUSSION TITLE
More details in menu options

### DISCUSSION THREAD

**Discussion by DanMan32** [mriscoc/Ender3V2S1#483] [Ideas]
This is a great firmware with lots of options and settings, many settings normally only available through recompiling with configuration file changes.

I'd like to see each menu item explained in better details.
Also for some settings, indicate if it can be saved to EEPROM or not.  For example Flow.  This option is available in at least two places, one I think under Control where you'd think everything here could be saved to non-volatile memory.
Perhaps tooltip when cursor is over a potential selection?

**Comment by mriscoc** [comment]
Not a firmware issue, moving to Discussions. Most of the options are explained on the Wiki.
